ui-tests/src/test/resources/features/technical-extension-integration.feature

```diff
@@ -89,18 +55,18 @@ Feature: Upload tech extension and add it to integration
     Then she is presented with a visual integration editor
     And she is prompted to select a "Start" connection from a list of available connections
 
+    # select postgresDB connection as 'from' point
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Periodic SQL Invocation" integration action
+    Then Camilla is presented with the Syndesis page "Periodic SQL Invocation"
+    Then she fills periodic query input with "SELECT * FROM CONTACT" value
+    Then she fills period input with "5000" value
+    And clicks on the "Done" button
+    Then she is prompted to select a "Finish" connection from a list of available connections
+
     # select twitter connection
     When Camilla selects the "Twitter Listener" connection
```

Comment:
The finish part of integration should be also changed to Invoke stored procedure `add_lead` and pls add simple mapping name to name.